modules/psap-installing-node-feature-discovery-operator.adoc

```diff
@@ -23,14 +23,17 @@ As a cluster administrator, you can install the NFD Operator using the CLI.
 
 . Create a namespace for the NFD Operator.
 
-.. Create the following `Namespace` custom resource (CR) that defines the `openshift-nfd` namespace, and then save the YAML in the `nfd-namespace.yaml` file:
+.. Create the following `Namespace` custom resource (CR) that defines the `openshift-nfd` namespace, and then save the YAML in the `nfd-namespace.yaml` file. Set cluster-monitoring to "true":
```

Comment:
Tiny nit, though I'm not sure - technically in the file, you use "true" with the double quotes. However, since that phrase (including the double quotes) is a value in the YAML file, I think you should probably wrap the entire thing in backticks:
```suggestion
.. Create the following `Namespace` custom resource (CR) that defines the `openshift-nfd` namespace, and then save the YAML in the `nfd-namespace.yaml` file. Set cluster-monitoring to `"true"`:
```